tests/phpunit/tests/widgets.php

```diff
@@ -1364,4 +1364,27 @@ public function test_sidebar_guessing_one_menu_per_sidebar() {
 		);
 		$this->assertSameSetsWithIndex( $expected_sidebars, $new_next_theme_sidebars );
 	}
+
+	/**
+	 * Make sure wp_map_sidebars_widgets always return an array of mapped sidebar widgets
+	 *
+	 * @covers ::wp_map_sidebars_widgets
+	 */
```

Comment:
It would be nice to mention the ticket ID here :)

Reply:
Done ;)